### ISSUE TITLE
ebook situation

### ISSUE THREAD

**Issue by cedounet** [hendricius/the-sourdough-framework#225]
Let's use that as a master bug.

There are several problems with our ebooks.

1. They are too big so nobody uses them on Kindle.  Theres is an action about that https://github.com/hendricius/the-sourdough-framework/issues/216
2. The whole (sans)-serif thing i think is useless.  From my experience the reader opens with last selected font...
3. We support 3 formats, two of them are outdated.  I have no problem keeping them, if they are actually used of course.
4. The list of flowchart does not work (despite 2 entries).  This might explain why you needed to do ruby magic for the website.
5. Some stuff don't show up greatly (degres, some tables, ...) 

Might sound like a rant, I promise it is not, just me not being a user of ebooks I need guidance on what makes sense to do and somebody testing it.

Opinions/thoughts ?

C/

**Comment by cedounet** [comment]
MWE for debug in https://github.com/hendricius/the-sourdough-framework/tree/225-ebook-situation

**Comment by hendricius** [comment]
1. I don't have a kindle myself. But my dad tried with the .epub version and he could use this tool to send it over (https://www.amazon.com/gp/sendtokindle). It supports max 200mb, so that should be good.

2. This was a request originally from someone who had a hart time reading the pdf with serifs. I am no expert on accessibility, but I propose we keep it for now?

3. Agreed - let's remove .mobi and azw3. We can update the instructions on the website for kindle users? It should save a lot of build time.

4. At least for the website it works currently.

5. Agreed. I will see if PR #223 fixes it and test a bit more.

**Comment by cedounet** [comment]
2) pdf sans serif makes sense.  Plenty of reasons not to like serif on low res screens… or other reasons.  We should keep it.

The sans serif ebook I am more puzzled about as ebook will choose the font it wants and it’s size.

**Comment by cedounet** [comment]
>  4. At least for the website it works currently.

Yes agree.. you did a good job at working around the issues for the website, just that does not solve the ebook.  And we should fix them at the root if we can :)

**Comment by cedounet** [comment]
> Agreed - let's remove .mobi and azw3. We can update the instructions on the website for kindle users? It should save a lot of build time.

I will do that in 2 steps... First remove downloads links and see if somebody is unhappy... If not I will remove the code in the makefile.

**Comment by cedounet** [comment]
> > Agreed - let's remove .mobi and azw3. We can update the instructions on the website for kindle users? It should save a lot of build time.
> 
> I will do that in 2 steps... First remove downloads links and see if somebody is unhappy... If not I will remove the code in the makefile.

https://github.com/hendricius/the-sourdough-framework/pull/227

**Comment by cedounet** [comment]
> 2. pdf sans serif makes sense.  Plenty of reasons not to like serif on low res screens… or other reasons.  We should keep it.
> 
> 
> The sans serif ebook I am more puzzled about as ebook will choose the font it wants and its size.

https://github.com/hendricius/the-sourdough-framework/pull/228

**Comment by hendricius** [comment]
@cedounet thanks! I agree, I checked on some ebook readers and they indeed ship their own fonts mostly. So it makes no sense to have 2 versions for epub. Only for the PDF.

**Comment by cedounet** [comment]
> The list of flowchart does not work (despite 2 entries). This might explain why you needed to do ruby magic for the website.

https://github.com/hendricius/the-sourdough-framework/pull/240

**Comment by cedounet** [comment]
Update:

- No flowcharts on ebook https://github.com/hendricius/the-sourdough-framework/issues/279
- symbols not showing up https://github.com/hendricius/the-sourdough-framework/issues/184 https://github.com/hendricius/the-sourdough-framework/issues/182
